docs/introduction/getting-started.mdx

```diff
@@ -28,20 +32,220 @@ To get started:
+tar zxvf conduit_0.8.0_Darwin_arm64.tar.gz
+```
+
+Depending on your operating system you may need to make the conduit binary executable.
+```shell
+chmod a+x conduit
 ```
 
-3. Start Conduit:
+## Starting Conduit
+Now that we have Conduit installed let's start it up to see what happens.
 
 ```shell
 ./conduit
 ```
 
-**Tip**: Depending on your operating system, you may need to
-run `chmod +x conduit` before running the binary.
+```shell
+             ....
+         .::::::::::.
+       .:::::‘‘‘‘:::::.
+      .::::        ::::.
+ .::::::::          ::::::::.
+ `::::::::          ::::::::‘
+      `::::        ::::‘
+       `:::::....:::::‘
+         `::::::::::‘        Conduit v0.8.0 darwin/arm64
+             ‘‘‘‘
+2024-02-21T14:41:26+00:00 INF All 0 tables opened in 0s component=badger.DB
+2024-02-21T14:41:26+00:00 INF Discard stats nextEmptySlot: 0 component=badger.DB
+2024-02-21T14:41:26+00:00 INF Set nextTxnTs to 0 component=badger.DB
+2024-02-21T14:41:26+00:00 INF builtin plugins initialized component=builtin.Registry count=6
+2024-02-21T14:41:26+00:00 WRN could not read plugin directory error="open /app/conduit/connectors: no such file or directory" component=standalone.Registry
+2024-02-21T14:41:26+00:00 INF standalone plugins initialized component=standalone.Registry count=0 plugin_path=/app/conduit/connectors
+2024-02-21T14:41:26+00:00 INF processors initialized component=processor.Service count=0
+2024-02-21T14:41:26+00:00 INF connectors initialized component=connector.Service count=0
+2024-02-21T14:41:26+00:00 INF pipelines initialized component=pipeline.Service count=0
+2024-02-21T14:41:26+00:00 INF pipeline configs provisioned component=provisioning.Service created=[] deleted=[] pipelines_path=./pipelines
+2024-02-21T14:41:26+00:00 INF grpc server started address=[::]:8084
+2024-02-21T14:41:26+00:00 INF http server started address=[::]:8080
+2024-02-21T14:41:26+00:00 INF
+2024-02-21T14:41:26+00:00 INF click here to navigate to Conduit UI: http://localhost:8080/ui
+2024-02-21T14:41:26+00:00 INF click here to navigate to explore the HTTP API: http://localhost:8080/openapi
+2024-02-21T14:41:26+00:00 INF
+```
+
+A few things to point out in this start up message.
 
-4. Navigate to `http://localhost:8080` to check Conduit's UI:
+1. We see conduit warning that there's no `connectors` directory. This is fine because we haven't created one yet.
+```shell
+2024-02-21T14:41:26+00:00 WRN could not read plugin directory error="open /app/conduit/connectors: no such file or directory" component=standalone.Registry
+```
+
+2. We see Conduit telling us the location of the Conduit UI and the HTTP API
+```shell
+2024-02-21T14:41:26+00:00 INF click here to navigate to Conduit UI: http://localhost:8080/ui
+2024-02-21T14:41:26+00:00 INF click here to navigate to explore the HTTP API: http://localhost:8080/openapi
+```
+
+Now that we have Conduit up and running you can now navigate to `http://localhost:8080` to check this admin UI:
```

Comment:
Maybe: this admin UI -> its admin UI or the admin UI?

Reply:
thanks!